benchmarks/src/tpch/run.rs

```diff
@@ -81,6 +84,10 @@ pub struct RunOpt {
     /// Whether to disable collection of statistics (and cost based optimizations) or not.
     #[structopt(short = "S", long = "disable-statistics")]
     disable_statistics: bool,
+
+    /// Whether to disable collection of statistics (and cost based optimizations) or not.
+    #[structopt(short = "j", long = "hash-join", default_value = "true")]
+    prefer_hash_join: BoolDefaultTrue,
```

Comment:
I am worried that this might switch the `tpch` runs to using SMJ by accident (given your comment above). I started some benchmark runs to see if we can get some data one way or the other

Reply:
Hash join enabled by default, SMJ is a separate key which is not even documented yet as it still in experimental

Reply:
benchmarks look good to me. ✅